react/render_server.py

```diff
@@ -39,11 +39,16 @@ def render(self, path, props=None, to_static_markup=False):
         serialized_options = json.dumps(options)
         options_hash = hashlib.sha1(serialized_options.encode('utf-8')).hexdigest()
 
+        request_headers = {'content-type': 'application/json'}
+
+        if headers is not None:
+            request_headers.update(headers)
```

Comment:
I would note somewhere that headers is a dictionary, as this line will blow up spectacularly if it isn't and there's no real documentation of it besides the error.